### PR TITLE
Setup Apollo GraphQL client with cache persisting

### PR DESCRIPTION
**Pull request by donni106**
- setup Apollo GraphQL client: https://www.apollographql.com/docs/react/
  - a local development address is given for the GraphQL server uri
- implemented cache persisting with https://github.com/apollographql/apollo-cache-persist
  to AsyncStorage
- made first queries
  - on HomeScreen querying data from locally connected main server
  - on IndexScreen querying data from the client store
  - for `fetchPolicy`s see https://www.apollographql.com/docs/react/api/react-apollo#graphql-config-options-fetchPolicy
- increased version to 0.2.0

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,86 @@
...
+    // Setup your Apollo Link, and any other Apollo packages here.
+    const client = new ApolloClient({
+      link,
+      cache,
+
+      // If you want to use Apollo Client's @client support to query the cache without using
+      // local resolvers, you must pass an empty object into the ApolloClient constructor
+      // resolvers option. Without this Apollo Client will not enable its integrated @client
+      // support, which means your @client based queries will be passed to the Apollo Client
+      // link chain. You can find more details about why this is necessary here.
```
Comment (SoleCincis) [inline src/index.js]: link is missing

File: src/screens/IndexScreen.js [hunk 2]
```diff
@@ -49,19 +32,55 @@ export default class IndexScreen extends React.Component {
...
+        {({ data, client }) => {
+          const { cacheItems } = data;
+
+          return (
+            <View style={styles.container}>
+              <Text>Index Screen</Text>
+              {!!cacheItems &&
+                cacheItems.map((item) => (
+                  <Button
+                    key={`bla${item.itemId}`}
```
Comment (SoleCincis) [inline src/screens/IndexScreen.js]: To do : bla needs to be changed with a appropriate word

File: src/index.js (hunk 1 above)
Comment (donni106) [reply src/index.js]: thx, added it

File: src/screens/IndexScreen.js (hunk 2 above)
Comment (donni106) [reply src/screens/IndexScreen.js]: thx, changed it to `key`.

**Pull request merged**